### PR TITLE
chore: setup file acting as centralized location for all colors

### PR DESCRIPTION
**Pull request by stefanomusicman**
# Description of the problem
created a file which is acting as a centralized location for all colors currently present on the application
organization style is based on pages of the application

## How does it solve the problem
if changes need to be made, it can be done in one place and changes will take effect all across the app

# References (also the Jira ticket)
BAR-11

# Checklist:

- [ ] I re-reviewed the Jira ticket now to make sure 
- [ ] My code follows the style guidelines of this project & of Dart
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code in hard-to-understand areas
- [ ] I have wrote some documentation if I changed a big behavior of the code
- [ ] My changes generate no new warnings/and if they do, I described them to the team
- [ ] I have added tests that prove my fix is effective or that my feature works if I worked on a logic class/function
- [ ] New and existing unit and integration tests pass locally with my changes

### CODE REVIEW

**Comment by danielemonaca** [comment]
I didn't think about this approach in terms of structuring the colors, _but I really like it_, however, I'm not a fan of code duplicates, and inheritance comes in support of it! 

Think about extracting things like `redBackground` and `whiteBackground` and any common colors to a parent class which the children(the screens) can inherit from.

Let me know if it makes sense, if not we can quickly make a pp-session.

File: lib/globals/theme/app_themes.dart [hunk 1]
```diff
@@ -0,0 +1,167 @@
+import 'package:flutter/material.dart';
+
+// each class will represent colors used in a specific page with the page title used in the class title
+
+// get started page
+class GetStartedColors {
```
Comment (danielemonaca) [inline lib/globals/theme/app_themes.dart:6]: I didn't think about this approach in terms of structuring the colors, _but I really like it_, however, I'm not a fan of code duplicates, and inheritance comes in support of it!

Think about extracting things like `redBackground` and `whiteBackground` and any common colors to a parent class which the children(the screens) can inherit from.

Let me know if it makes sense, if not we can quickly make a pp-session.

**Comment by danielemonaca** [comment]
**About failings checks:**

Let's go into details
image

and we check what is written here:
image

As it is written here, it is not the test failing but the formatting, to fix it, it should be enough to just run `flutter format` in your repo and then push the new formatted changes


**About `This branch is out-of-date with the base branch`:**

This is totally normal and we will encounter it most of the time because we are working together in the same codebase.
As yesterday the CI created some commits on main to add the workflows files (under .github), it updated main and you, locally didn't have the most updated version.

In this case, just clicking the `Update branch button` would fix everything 
image

Sometimes, however, `conflicts` will come up, and that is also normal, but it is then probably better and safer to fix inside of your repo instead of from GitHub.

**Comment by stefanomusicman** [comment]
ok perfect i'll make the changes

**Review by danielemonaca — APPROVED** [review]

**Comment by danielemonaca** [comment]
Done, what I have done is:

1. run `flutter format lib` that actually discovered that our `main.dart` was using a deprecated `heading4` property, I now changed it to `headingMedium` which is the updated version.  That fixed the flutter format job in the pipeline.
2. It was still complaining about the `flutter analyze`, to fix that, I just merge the latest main with your previous PR and it fixed automatically.

Feel free to merge!

**Pull request merged**